lardata/ArtDataHelper/Dumpers/dump_pids.fcl

```diff
@@ -0,0 +1,79 @@
+#
+# File:     DumpParticleIDs.fcl
+# Purpose:  Dump on screen particle ID content
+# Author:   H. Greenlee
+# Date:     Oct. 14, 2021
+# Version:  1.0
+#
+# Service dependencies:
+# - message facility
+#
+
+process_name: DumpParticleIDs
+
+services: {
+  
+  message: {
+  #   debugModules: [ "*" ]
+    destinations: {
+      
+      # grab all the "DumpParticleIDs" messages and put them in DumpPIDs.log
+      DumpParticleIDs: {
+        append: false
+        categories: {
+          DumpParticleIDs: { limit: -1 }
+          default: { limit: 0 }
+        }
+        filename: "DumpPIDs.log"
+        threshold: "INFO"
+        type: "file"
+      } # DumpParticleIDs
+      
+      LogStandardOut: {
+        categories: {
+          AnaBaseDefaultCtor: { limit: 0 }
+          GeometryBadInputPoint: {
+            limit: 5
+            timespan: 1000
+          }
+          RecoBaseDefaultCtor: { limit: 0 }
```

Comment:
I have not noticed any need for this in years... I recommend removal.
```suggestion
```

Reply:
Removed.